code/game/machinery/computer/medical.dm

```diff
@@ -40,7 +40,7 @@
 	var/list/data = list()
 
 	var/has_access = authenticated && isliving(user)
-	data["authenticated"] = authenticated
+	data["authenticated"] = authenticated || isAdminGhostAI(user)
```

Comment:
Would this work since ``has_access`` would still be false?

Reply:
good catch